test/jest.setup.js

```diff
@@ -25,3 +26,29 @@ global.cozy = {
     setTheme: () => null
   }
 }
+
+const ignoredWarnings = {
+  I18n: {
+    reason: 'Preact compatibility',
+    matcher: makeDeprecatedLifecycleMatcher('I18n')
+  },
+  ModalContent: {
+    reason: 'Preact compatibility',
+    matcher: makeDeprecatedLifecycleMatcher('ModalContent')
+  },
+  ModalAriaLabel: {
+    reason: 'Dont know',
+    matcher: msg => msg.includes('aria-label') && msg.includes('If your modal')
```

Comment:
I think this would hide real problems: cozy-ui's modal requires to have an aria-label for accessibility reasons. We should add missing aria-labels on our modals.

Reply:
OK fair enough, the problem is that the Modal issues this warning even if we have `aria-label`. I am on it. https://github.com/cozy/cozy-ui/issues/1338

Reply:
:+1: